README.md
# Liberation
[![CI](https://github.com/7Cav/Liberation/actions/workflows/main.yml/badge.svg?branch=develop)](https://github.com/7Cav/Liberation/actions/workflows/main.yml)

7Cav variation of KP Liberation for Arma3


CHANGELOG.md
# Changelog
## 0.96.7a-7cav-1.01
* Added: Blacklisted ACE Vehicle Keys & Deadman's Switch.
* Tweaked: CBA settings to align with those used by cScripts.

## 0.96.7a-7cav-0.8
* Fixed: Removed fire barrels from buildable due to bug with DLC
* Fixed: Issue with UH-34/CH-47 cargo slot increase not persisting after server restart
* Tweaked: Disable ACE cookoff
* Tweaked: Reduced chance that enemy will surrender, please be less soi.
* Tweaked: increased FOB build range to 200m.
* Tweaked: increased civ kill penalty to 5%.
* Tweaked: Default weather is sunny.

## 0.96.7a-7cav-0.7.1
* Fixed: Allow everyone to use epinephrine

## 0.96.7a-7cav-0.7
* Tweaked: Increased ace cargo space to 40 for the CH-47 and UH-34
* Tweaked: Cleaned up all map SQMs
* Tweaked: Adjusted permissions for adminlogged zeus
* Tweaked: Reduced chance of snake bite
* Fixed: Infantry being ejected from doorgunner/crew chief seats
* Fixed: Missing helicopter helemts on Cam Lao Nam
* Fixed: Missing rifle grenades on Cam Lao Nam
* Fixed: Reduced refresh time on Production menu
* Fixed: Removed AI support system not in use

## 0.96.7a-7cav-0.6
* Added: Respawning UH-1D on 10 minute delay to Operation Base on Cam Lao Nam
* Added: Buildable lamp to Cam Lao Nam
* Added: Hostile fob templates to Cam Lao Nam
* Added: Buildable ammo and medical crates
* Fixed: Pilots not having engineer perms on Cam Lao Nam
* Fixed: Airport Refuel/Repair/Rearm vehicles being airliftable - These are just for the master arm functionality at base not to be moved to fobs
* Fixed: Vanilla FAKs not being converted to ACE bandages
* Fixed: Missing M-14 scopes in arsenal on Cam Lao Nam
* Fixed: Implemented potential fix for bugged AOs on Cam Lao Nam
* Fixed: 105MM Arty missing from build menu on Cam Lao Nam
* Fixed: Ignored buildings wasn't enabling properly on Chernarus
* Fixed: Incorrect variable name in manage_logistics.sqf
* Tweaked: Blufor/Opfor/Civilian/Resistance presets on Cam Lao Nam
* Tweaked: Increased supply crate value paramater
* Tweaked: Further performance optimizations
* Tweaked: Reenabled mobile arsenal buildable via build menu
* Tweaked: Convert all .HPP files into .INC
* Tweaked: Lowered duration and probability of cookoff
* Removed: Mousewheel earplugs option and CDLC gesture menu on Cam Lao Nam

## 0.96.7a-7cav-0.5.1
* Added: CDLC Master Arm functionality to operation base
* Added: Additional start vehicles including PT boats and more Hueys on Cam Lao Nam
* Added: Server rules billboard to base on Cam Lao Nam
* Fixed: Further adjust slots to cav standards on Cam Lao Nam
* Fixed: Messages diplaying for teamkills/civilian kills/resistance kills/epw kills
* Fixed: 343s being given on respawn on Cam Lao Nam
* Fixed: Multiple performance enhancements to various systems
* Fixed: Changed models for pilots on Cam Lao Nam to prevent futuristic underwear display
* Fixed: Air vehicle build list to include all US Army air vehicles and Navy/Air Force variants of F4
* Fixed: No vehicles being locked behind military bases

## 0.96.7a-7cav-0.5 (24HR OP)
* Added: Ace Rearm,Repair,Medical to applicable supply vehicles for SOGPF CDLC.
* Added: Arsenal Entries for M16, M1 Carbine, ..etc Scopes and Bayonets.
* Added: Function to remove ACRE radios for client_init
* Fixed: Slots for Cam_Lao_Nam Map to be more appropriated for 7cav use.
* Fixed: PRC343 Spawning on init.
* Removed: That stupid buildable bar with questionable name
* Tweaked: Moved startbase to inside the base
* Tweaked: Added more hueys to start to support buffalo slots
* Tweaked: Changed slot layout for 24 hour op

## 0.96.7a-7cav-0.4
* Fixed: Changed Raider-1 from helipilot model to helicrew to prevent CAS locking out HALO jump
* Fixed: Made GUNSLINGER groups infantry, modified one group to VIKING for mechanized
* Fixed: Reordered mission.sqm after group changes
* Fixed: Moved flag at Takistan operation base to prevent grizzly losing rotors when using Advanced Flight Model
* Added: FRIES functionality to UH-60M Desert variant
* Added: F/A-181 Black Wasp II spawns to carriers to give early fixed wing CAS
* Added: Map support for Tanoa
* Added: Map support for Chernarus(Winter)
* Added: Ported mission to Everon
* Added: Teamkills, EPW kills, Civilian kills, and Civilian vehicle kills will now announce the killer's name in chat
* Added: UH-60M to operation base spawn
* Added: Cav Flag to operation base
* Added: Rules Splash Screen on Mission Start (thanks to Fox.S for the hard work on the graphic)
* Tweaked: Platoon Staff now have own groups for BFT Readability
* Tweaked: Changed Buffalo-2 to Raven-1 and Raven-2
* Tweaked: Made Infidel-1 its own group and added an observer
* Tweaked: Changed second SABER group to BANSHEE
* Tweaked: Changed names of groups for consistency with cav standards
* Removed: Removed one little bird spawn from operation base
* Tweaked: Changed Killah Potatoes flag to Cav Flag in build menu
* Tweaked: Blacklisted meme stuff from arsenal
* Tweaked: Broke abandoned blufor vehicle markers out of map marker parameter to allow visibility without enabling all map markers

## 0.96.7a-7cav-0.3 (2021-05-03)
* Added: Block for non-pilots entering buildable air vehicles (You must be in pilot spot to pilot a vehicle)
* Tweaked: CBA_Settings for ACE_Medical to be more forgiving due to infrequent support assets (Less fractures, Less instant bleedouts).
* Tweaked: ACRE Radios to ARCADE Mode, So I maintain sanity when someone tells me that ACRE is broken cause they can't comprehend 20km range LOS.
* Tweaked: ACE_Settings so ballistics is disabled. Mainly so people stop telling me its the reason the server runs slow.
* Fixed: Issue with mission default parameters that caused CUP Maps to think a bullet hole is a destroyed building and thus tanking civ reputation.
* Fixed: Parameter setting that adjusts the AI count to Player Count, which sound great until you have 350 units and 10 SFPS.
* Fixed: Unit Cap settings to allow for less overhead until Tact 2 is configured properly.

Missionframework/ui/liberation_titles.hpp
class CfgSounds {
    sounds[] = {};
    class parasound	{
        name = "parasound";
        sound[] = {"res\c130.ogg", 1.0, 1};
        titles[] = {};
    };

    class oof
	{
		name = "oof";
		sound[] = { "audio\oof.ogg", 1, 1, 100 };
		titles[] = {};
	};
};

class RscTitles {
    class intro1 {
        name = "intro1";
        duration = 4;
        idd = -1;
        movingEnable = false;
        controls[]= {
            GenericLabelShadow, GenericLabel2, GenericLabel3
        };
        class GenericLabel2 {
            idc = -1;
            type =  CT_STATIC ;
            style = ST_CENTER;
            colorText[] = COLOR_WHITE;
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.035 * safezoneH;
            x = 0.3 * safezoneW + safezoneX;
            w = 0.4 * safezoneW;
            y = 0.65 * safezoneH + safezoneY;
            h = 0.1 * safezoneH;
            text = "[GREUH] KP and =7Cav= S3 Department presents";
            shadow = 1;
        };
        class GenericLabelShadow: GenericLabel2 {
            shadow = 2;
        };
        class GenericLabel3 {
            idc = -1;
            type =  CT_STATIC ;
            style = ST_CENTER;
            colorText[] = COLOR_WHITE;
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.025 * safezoneH;
            x = 0.3 * safezoneW + safezoneX;
            w = 0.4 * safezoneW;
            y = 0.70 * safezoneH + safezoneY;
            h = 0.1 * safezoneH;
            text = "Special Thanks to Sweetwater.I, Liber.N and Sypolt.R";
            shadow = 1;
        };
    };
    class intro2 {
        name = "intro2";
        duration = 7;
        idd = -1;
        movingEnable = false;
        controls[] = {
            VersionLabelShadow, Splash, VersionLabel
        };
        class Splash {
            idc = -1;
            type =  CT_STATIC ;
            style = ST_PICTURE;
            colorText[] = COLOR_WHITE;
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.1 * safezoneH;
            x = 0.325 * safezoneW + safezoneX;
            w = 0.35 * safezoneW;
            y = 0.2 * safezoneH + safezoneY;
            h = 0.6 * safezoneH;
            text = "res\splash_libe2.paa";
        };
        class VersionLabel {
            idc = -1;
            type =  CT_STATIC ;
            style = ST_CENTER;
            shadow = 1;
            colorText[] = COLOR_WHITE;
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.035 * safezoneH;
            x = 0.45 * safezoneW + safezoneX;
            w = 0.3 * safezoneW;
            y = 0.65 * safezoneH + safezoneY;
            h = 0.1 * safezoneH;
            text = $STR_MISSION_VERSION;
        };
        class VersionLabelShadow: VersionLabel {
            shadow = 2;
            font = FontM;
        };
    };
    class fasttravel {
        name = "fasttravel";
        duration = 4;
        idd = -1;
        movingEnable = true;
        controls[] = {
            OuterBackground,GenericLabel111
        };
        class OuterBackground {
            idc = -1;
            type =  CT_STATIC ;
            style = ST_LEFT;
            colorText[] = COLOR_BLACK;
            colorBackground[] = COLOR_BLACK;
            font = FontM;
            sizeEx = 0.023;
            x = -3; y = -3;
            w = 9;  h = 9;
            text = "";
        };
        class GenericLabel111 {
            idc = -1;
            type =  CT_STATIC ;
            style = ST_CENTER;
            colorText[] = COLOR_WHITE;
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.03;
            x = 0; y = 0.75;
            w = 1.0;  h = 0.1;
            text = $STR_DEPLOY_IN_PROGRESS;
        };
    };
    class KPLIB_overlay {
        name = "KPLIB_overlay";
        duration = 999999;
        idd = 6666;
        movingEnable = true;
        onLoad = "uiNamespace setVariable ['KPLIB_overlay', _this select 0];";
        controls[] = {
            BGPicture,
            PictureFOBShadow, PictureFOB, LabelFOB,
            PictureSuppliesShadow, PictureSupplies, LabelSupplies,
            PictureAmmoShadow, PictureAmmo, LabelAmmo,
            PictureFuelShadow, PictureFuel, LabelFuel,
            PictureCapShadow, PictureCap, LabelCap,
            PictureHeliShadow, PictureHeli, LabelHeli,
            PicturePlaneShadow, PicturePlane, LabelPlane,
            PictureCombatReadinessShadow, PictureCombatReadiness, LabelCombatReadiness,
            PictureCivRepShadow, PictureCivRep, LabelCivRep,
            PictureIntelShadow, PictureIntel, LabelIntel,
            BGPictureActiveSectors, BGPictureSector,
            CaptureFrame_OPFOR, CaptureFrame_BLUFOR, CaptureFrame,
            LabelPoint, CentralShadow, CentralLabel,
            ActiveSectors, AlertBGPicture, AlertLabel, AlertTimer
        };
        class BGPicture {
            idc = 758001;
            type =  CT_STATIC;
            style = ST_PICTURE;
            colorText[] = {0.8,0.8,0.8,0.9};
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.5;
            x = 0.95 * safezoneW + safezoneX;
            w = 0.075 * safezoneW;
            y = 0.395 * safezoneH + safezoneY;
            h = 0.255 * safezoneH;
            text = "res\gradient.paa";
        };
        class BGPictureSector: BGPicture {
            idc = 201;
            x = 0.9 * safezoneW + safezoneX;
            w = 0.15 * safezoneW;
            y = 0.33 * safezoneH + safezoneY;
            h = 0.05 * safezoneH;
        };
        class GenericLabel69 {
            idc = -1;
            type =  CT_STATIC;
            style = ST_RIGHT;
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.022 * safezoneH;
            x = 0.905 * safezoneW + safezoneX;
            w = 0.08 * safezoneW;
            y = 0.5225 * safezoneH + safezoneY;
            h = 0.04 * safezoneH;
            text = "";
            shadow = 2;
        };
        class CentralLabel: GenericLabel69 {
            idc = 266;
            type = CT_STATIC;
            style = ST_CENTER;
            colorText[] = {1,1,0,1};
            sizeEx = 0.03 * safezoneH;
            x = 0 * safezoneW + safezoneX;
            w = 1 * safezoneW;
            y = 0.2 * safezoneH + safezoneY;
            h = 0.05 * safezoneH;
            text = "";
        };
        class CentralShadow: CentralLabel {
            idc = 267;
            shadow = 1;
        };
        class GenericPicture69 {
            idc = -1;
            type =  CT_STATIC;
            style = ST_PICTURE;
            colorText[] = {0.9,0.9,0.9,1};
            colorBackground[] = COLOR_NOALPHA;
            font = FontM;
            sizeEx = 0.5;
            x = 0.985 * safezoneW + safezoneX;
            w = 0.012 * safezoneW;
            h = 0.0213333333 * safezoneH;
        };
        class PictureFOB: GenericPicture69 {
            idc = 758002;
            y = (0.4 + ( ICONE_SPACY * 0 ) ) * safezoneH + safezoneY;
            text = "\A3\ui_f\data\map\markers\handdrawn\flag_CA.paa";
        };
        class PictureFOBShadow: PictureFOB {
            idc = 758003;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 0 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelFOB: GenericLabel69 {
            idc = 758004;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 0 ) ) * safezoneH + safezoneY;
            colorText[] = {0.8, 0.8, 0.8, 1};
        };
        class PictureSupplies: GenericPicture69 {
            idc = 758005;
            y = (0.4 + ( ICONE_SPACY * 1 ) ) * safezoneH + safezoneY;
            text = "res\ui_manpo.paa";
        };
        class PictureSuppliesShadow: PictureSupplies {
            idc = 758006;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 1 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelSupplies: GenericLabel69 {
            idc = 758007;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 1 ) ) * safezoneH + safezoneY;
            colorText[] = {0, 0.75, 0, 1};
        };
        class PictureAmmo: GenericPicture69 {
            idc = 758008;
            y = (0.4 + ( ICONE_SPACY * 2 ) ) * safezoneH + safezoneY;
            text = "res\ui_ammo.paa";
        };
        class PictureAmmoShadow: PictureAmmo {
            idc = 758009;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 2 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelAmmo: GenericLabel69 {
            idc = 758010;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 2 ) ) * safezoneH + safezoneY;
            colorText[] = {0.75, 0, 0, 1};
        };
        class PictureFuel: GenericPicture69 {
            idc = 758011;
            y = (0.4 + ( ICONE_SPACY * 3 ) ) * safezoneH + safezoneY;
            text = "res\ui_fuel.paa";
        };
        class PictureFuelShadow: PictureFuel {
            idc = 758012;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 3 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelFuel: GenericLabel69 {
            idc = 758013;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 3 ) ) * safezoneH + safezoneY;
            colorText[] = {0.75, 0.75, 0, 1};
        };
        class PictureCap: GenericPicture69 {
            idc = 758014;
            y = (0.4 + ( ICONE_SPACY * 4 ) ) * safezoneH + safezoneY;
            text = "\a3\Ui_F_Curator\Data\Displays\RscDisplayCurator\modeGroups_ca.paa";
        };
        class PictureCapShadow: PictureCap {
            idc = 758015;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 4 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelCap: GenericLabel69 {
            idc = 758016;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 4 ) ) * safezoneH + safezoneY;
            colorText[] = {0.8, 0.8, 0.8, 1};
        };
        class PictureHeli: GenericPicture69 {
            idc = 758017;
            y = (0.4 + ( ICONE_SPACY * 5 ) ) * safezoneH + safezoneY;
            text = "\A3\air_f_beta\Heli_Transport_01\Data\UI\Map_Heli_Transport_01_base_CA.paa";
        };
        class PictureHeliShadow: PictureHeli {
            idc = 758018;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 5 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelHeli: GenericLabel69 {
            idc = 758019;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 5 ) ) * safezoneH + safezoneY;
            colorText[] = {0.8, 0.8, 0.8, 1};
        };
        class PicturePlane: GenericPicture69 {
            idc = 758020;
            y = (0.4 + ( ICONE_SPACY * 6 ) ) * safezoneH + safezoneY;
            text = "\A3\Air_F_EPC\Plane_CAS_01\Data\UI\Map_Plane_CAS_01_CA.paa";
        };
        class PicturePlaneShadow: PicturePlane {
            idc = 758021;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 6 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelPlane: GenericLabel69 {
            idc = 758022;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 6 ) ) * safezoneH + safezoneY;
            colorText[] = {0.8, 0.8, 0.8, 1};
        };
        class PictureCombatReadiness: GenericPicture69 {
            idc = 758023;
            y = (0.4 + ( ICONE_SPACY * 7 ) ) * safezoneH + safezoneY;
            text = "\A3\ui_f\data\map\markers\handdrawn\warning_CA.paa";
        };
        class PictureCombatReadinessShadow: PictureCombatReadiness {
            idc = 758024;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 7 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelCombatReadiness: GenericLabel69 {
            idc = 758025;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 7 ) ) * safezoneH + safezoneY;
            colorText[] = {0.8, 0.8, 0.8, 1};
        };
        class PictureCivRep: GenericPicture69 {
            idc = 758026;
            y = (0.4 + ( ICONE_SPACY * 8 ) ) * safezoneH + safezoneY;
            text = "\A3\ui_f\data\map\mapcontrol\tourism_CA.paa";
        };
        class PictureCivRepShadow: PictureCivRep {
            idc = 758027;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 8 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelCivRep: GenericLabel69 {
            idc = 758028;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 8 ) ) * safezoneH + safezoneY;
            colorText[] = {0.8, 0.8, 0.8, 1};
        };
        class PictureIntel: GenericPicture69 {
            idc = 758029;
            colorText[] = {0,0.45,0.95,1};
            y = (0.4 + ( ICONE_SPACY * 9 ) ) * safezoneH + safezoneY;
            text = "\A3\Ui_f\data\GUI\Cfg\Ranks\general_gs.paa";
        };
        class PictureIntelShadow: PictureIntel {
            idc = 758030;
            y = (0.4 + SHADOW_Y + ( ICONE_SPACY * 9 ) ) * safezoneH + safezoneY;
            x = (0.985 + SHADOW_X) * safezoneW + safezoneX;
            colorText[] = {0,0,0,0.7};
        };
        class LabelIntel: GenericLabel69 {
            idc = 758031;
            text = "";
            y = (0.39 + ( ICONE_SPACY * 9 ) ) * safezoneH + safezoneY;
            colorText[] = {0, 0.45, 0.95, 1};
        };
        class CaptureFrameStandard {
            idc = -1;
            type =  CT_STATIC;
            font = FontM;
            sizeEx = 0.023;
            text = "";
        };
        class CaptureFrame: CaptureFrameStandard {
            idc = 202;
            style = ST_FRAME;
            colorText[] = COLOR_BLACK;
            colorBackground[] = COLOR_OPFOR_NOALPHA;
            x = 0.9125 * safezoneW + safezoneX;
            w = 0.085 * safezoneW;
            y = 0.358 * safezoneH + safezoneY;
            h = 0.012 * safezoneH;
        };
        class CaptureFrame_OPFOR: CaptureFrameStandard{
            idc = 203;
            style = ST_TYPE;
            colorText[] = {0.6, 0, 0, 1};
            colorBackground[] = {0.6, 0, 0, 1};
            x = 0.9125 * safezoneW + safezoneX;
            w = 0.084 * safezoneW;
            y = 0.358 * safezoneH + safezoneY;
            h = 0.011 * safezoneH;
        };
        class CaptureFrame_BLUFOR: CaptureFrameStandard{
            idc = 244;
            style = ST_TYPE;
            colorText[] = {0, 0.2, 0.6, 1};
            colorBackground[] = {0, 0.2, 0.6, 1};
            x = 0.9125 * safezoneW + safezoneX;
            w = 0.084 * safezoneW;
            y = 0.358 * safezoneH + safezoneY;
            h = 0.011 * safezoneH;
        };
        class LabelPoint: GenericLabel69 {
            idc = 205;
            text = "";
            style = ST_RIGHT;
            sizeEx = 0.024 * safezoneH;
            x = 0.8 * safezoneW + safezoneX;
            w = 0.2 * safezoneW;
            y = 0.33 * safezoneH + safezoneY;
            h = 0.03 * safezoneH;
            colorText[] = {0.7, 0, 0, 1};
        };
        class ActiveSectors	{
            idc = 516;
            type = CT_STRUCTURED_TEXT;
            colorBackground[] = COLOR_NOALPHA;
            style = ST_RIGHT;
            x = 0.7 * safezoneW + safezoneX;
            w = 0.295 * safezoneW;
            y = 0.8 * safezoneH + safezoneY;
            h = 0.2 * safezoneH;
            text= "";
            size = 0.02 * safezoneH;
            sizeEx = 0.02 * safezoneH;
            shadow = 2;
            font = FontM;
            color = "#e0e000";
            align = "right";
            valign = "top";
        };
        class BGPictureActiveSectors: BGPicture {
            idc = 517;
            x = 0.93 * safezoneW + safezoneX;
            w = 0.1 * safezoneW;
            y = 0.8 * safezoneH + safezoneY;
            h = 0.2 * safezoneH;
        };
        class AlertBGPicture: BGPicture {
            idc = 401;
            x = 0.9 * safezoneW + safezoneX;
            w = 0.15 * safezoneW;
            y = 0.27 * safezoneH + safezoneY;
            h = 0.05 * safezoneH;
        };
        class AlertLabel: GenericLabel69 {
            idc = 402;
            text = "";
            style = ST_CENTER;
            sizeEx = 0.022 * safezoneH;
            x = 0.9 * safezoneW + safezoneX;
            w = 0.1 * safezoneW;
            y = 0.2675 * safezoneH + safezoneY;
            h = 0.03 * safezoneH;
            colorText[] = {0.85, 0, 0, 1};
        };
        class AlertTimer: GenericLabel69 {
            idc = 403;
            text = "";
            style = ST_CENTER;
            sizeEx = 0.026 * safezoneH;
            x = 0.9 * safezoneW + safezoneX;
            w = 0.1 * safezoneW;
            y = 0.29 * safezoneH + safezoneY;
            h = 0.03 * safezoneH;
            colorText[] = {0.85, 0, 0, 1};
        };
    };
};
